### ISSUE TITLE
Consider project-specific ivy-todo-file option

### ISSUE THREAD

**Issue by waymondo** [Kungsgeten/ivy-todo#3]
Thanks for this great package.

In my setup, I like my todo files to be in the foot directory of each project. I was able to easily add the behavior locally with advice:

```
(defun ivy-todo-use-local-project-todo-file (orig-fun &rest args)
    (let ((ivy-todo-file (expand-file-name "ivy-todo.org" (projectile-project-root))))
      (apply orig-fun args)))
(advice-add 'ivy-todo :around #'ivy-todo-use-local-project-todo-file)
```

I'm wondering if this behavior should be a built-in option to `ivy-todo`. What do you think?

**Comment by Kungsgeten** [comment]
Hi! Great idea! Perhaps there should be a function named `ivy-todo-get-todo-file` or similar, which returns the path to `ivy-todo-file`. Then that function could be redefined by the user. Another solution is ofcourse to do it like you've done :)

**Comment by waymondo** [comment]
Yeah, I think using a getter function to get the file location would be enough to accomplish the behavior cleanly and probably the easiest solution?
